provision.sh

```diff
@@ -5,30 +5,35 @@ export DEBIAN_FRONTEND=noninteractive
 sudo add-apt-repository ppa:ondrej/php
 echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 
-sudo apt-get update && sudo apt-get upgrade
+sudo apt-get update -q && sudo apt-get upgrade -q
 
 # Force a blank root password for mysql
 echo "mysql-server mysql-server/root_password password " | debconf-set-selections
 echo "mysql-server mysql-server/root_password_again password " | debconf-set-selections
 
 # Install mysql, nginx, php8.0-fpm
-sudo apt-get install -q -y -f git mysql-server mysql-client nginx php8.0 php8.0-fpm php8.0-xdebug
+sudo apt-get install -q -y -f --no-install-recommends git mysql-server mysql-client nginx php8.0 php8.0-fpm php8.0-xdebug
 
 # Install commonly used php packages
-sudo apt-get install -q -y -f php8.0-curl php8.0-cli php8.0-mysqli php8.0-gd php8.0-dom php8.0-zip php8.0-mbstring php8.0-intl php8.0-redis
+sudo apt-get install -q -y -f php8.0-curl php8.0-cli php8.0-mysql php8.0-gd php8.0-zip php8.0-mbstring php8.0-intl php8.0-redis php8.0-xml
 
-sudo apt-get upgrade libpcre3
+sudo apt-get upgrade -q libpcre3
 
-sudo apt-get -y install curl dirmngr apt-transport-https lsb-release ca-certificates unzip
+# install nodejs and yarn
+sudo apt-get -y -q install curl dirmngr apt-transport-https lsb-release ca-certificates unzip
 curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
-sudo apt-get install -y nodejs
+sudo apt-get install -y -q nodejs
 npm install --global yarn
+cd /var/www/etoa && yarn install --frozen-lockfile
+
+# Install PHP composer dependencies
+cd /var/www/etoa && export COMPOSER_ALLOW_SUPERUSER=1;php composer.phar install --no-interaction
```

Comment:
`composer install` muss vor `yarn install` passieren, weil yarn referenziert files, die mit Composer installiert werden `error Package "" refers to a non-existing file '"/var/www/etoa/vendor/symfony/ux-live-component/assets"'.`

Reply:
Die `# Setup dummy client files` sollten auch vor Composer install erstellt werden, damit sicher gestellt wird, dass `manifest.json` und `entrypoints.json` bei `cache:clear/cache:warmup` vorhanden sind.

Reply:
https://github.com/etoa/etoa-gui/pull/501/commits/db413355535bcca7c06fb1d135965a998cc9afc0
and
https://github.com/etoa/etoa-gui/pull/501/commits/46635cd50baac580097a46b9e9dbb87ec760d8ea
should fix this